### ISSUE TITLE
Verifying Install - Updating Loop

### ISSUE THREAD

**Issue by Cosmas47** [ValveSoftware/SteamOS#385]
I had 145 on my system for the longest time, then I booted up and let it upgrade to the latest version. I was then caught in some sort of updating checking loop whereby it shows the Verifying ... and Updating... splash screen infinitely. 

Assuming the worst I formatted the drive, downloaded latest version and reinstalled. Same thing happens, but only *after* its able to do an initial 200Mb update at the Debian desktop level. IceWeasel can even access the web. So it's not network related? I thought. Well...

Nearest as I can figure it, somehow the update beyond this screws up eth0. When I boot into recovery mode I see 'Could Not Resolve repo.steampowered.com' at the end of the output and ping, apt-get update both fail. ifconfig shows no eth0 interface, just the loopback. Well how can there be no eth0 interface if I just performed an update before?

That's all I have to go on.

**Comment by sharkwouter** [comment]
The could not resolve error happens on boot with every SteamOS system, which happen because Nerworkmanager haan't started yet.

**Comment by Cosmas47** [comment]
@sharkwouter Makes sense, but I'm not sure where to go from here. It's always going to fail checking an update if NM hasn't started. What else could cause a loop like this?

**Comment by sharkwouter** [comment]
It could be something related to unattended updates, but it could also potentially be a hardware issue. It's hard to say without logs.

If you press the reset button when it freezes, does it boot?

**Comment by Cosmas47** [comment]
It never 'freezes' in the traditional sense. I'm stuck in a loop every time I boot the system. I can never get past the Updating/Verifying splash screen. Is there a log I can look into from recovery mode?

**Comment by Cosmas47** [comment]
I got it to get past the screen finally but I'm not 100% sure how. One of two things I did made it work:

1. Reset BIOS (Complete battery removal and wait 30 sec)
2. Removed a Hauppage HDTV Capture card from the system. (Driver issue?)

Now the system starts up though Steam wont immediately make a connection to the network. I guess it has to sleep some 30-60 seconds before NetworkManager actually kicks in. I can also reset the connection in the Network config menu and that works, though I have to do this every time.
